packages/adapter-hypertune/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # @flags-sdk/hypertune
 
+## 0.3.0
+
+- Nested flags are now supported.
+- A `key` is now passed to `createSource` to prevent conflicts with your `getHypertune` setup.
+
```

Comment:
@miraan can you run "pnpm changeset" at the root level instead, and undo these changes? generate the changelog automatically using changesets